.github/workflows/copilot-inactive-reminder.yml

```
@@ -0,0 +1,131 @@
+      - name: Check last GitHub Copilot activity
+        id: check-last-activity
+        run: |
+          echo "🔍 Checking Copilot license usage for Luno organization..."
+          
+          # List all GitHub Copilot seat assignments for the organization
+          RESPONSE=$(gh api \
+            -H "Accept: application/vnd.github+json" \
+            -H "X-GitHub-Api-Version: 2022-11-28" \
+            -H "Authorization: Bearer ${{ secrets.COPILOT_UPDATE_TOKEN }}" \
+            /orgs/luno/copilot/billing/seats)
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)